daemon/k8s_watcher.go

```diff
@@ -556,7 +556,7 @@ func (d *Daemon) endpointDelFn(obj interface{}) {
 func areIPsConsistent(ipv4Enabled, isSvcIPv4 bool, svc types.K8sServiceNamespace, se *types.K8sServiceEndpoint) error {
 	if isSvcIPv4 {
 		if !ipv4Enabled {
-			return fmt.Errorf("Received an IPv4 kubernetes service but IPv4 is "+
+			return fmt.Errorf("Received an IPv4 k8s service but IPv4 is "+
```

Comment:
I'm still struggling with this error a bit in general. It's obviously unrelated to this PR. I think we should still configure IPv4 services even if disabled and only ignore them in the datapath. That will allow to enable/disable IPv4 on the fly in the future. I'll file an issue to clean this up.